fem/gridfunc.hpp

```diff
@@ -340,6 +338,8 @@ class GridFunction : public Vector
 
    void GetVectorGradient(ElementTransformation &tr, DenseMatrix &grad) const;
 
+   void GetVectorGradientHat(ElementTransformation &T, DenseMatrix &gh) const;
```

Comment:
If we make this public we should also document it (and perhaps GetVectorGradient so that people understand the distinction)

Reply:
These comments are a nice addition!

I think both should be marked `@brief`